draft-ietf-scitt-architecture.md

```diff
@@ -421,6 +438,14 @@ See Subject under {{terminology}} Terminology.
 
 Issuers MAY use different signing keys (identified by `kid` in the resolved key manifest) for different Artifacts, or sign all Signed Statements under the same key.
 
+### Support for Multiple Statements
+
+An Issuer can make multiple Statements about the same Artifact. For example, an Issuer can make amended Statements about the same Artifact as their view changes over time.
+
+Multiple Issuers can make different, even conflicting Statements, about the same Artifact. Verifiers can choose which Issuers they trust.
```

Comment:
```suggestion
Multiple Issuers can make different, even conflicting Statements, about the same Artifact.
Verifiers can choose which Issuers they trust.
```

Reply:
Moved to https://github.com/ietf-wg-scitt/draft-ietf-scitt-architecture/pull/153.

Still, I prefer not to have a newline here. To me, the sentence "Verifiers can choose which Issuers they trust" is closely linked to the previous "conflicting Statements". When on its own paragraph, this sentence seems to be coming out of the blue, with little context.

Reply:
Note, the newline is PR feedback friendly, and markdown will put this on the same line.

Reply:
Ah oops, I thought there was a blank line inbetween. Fixed, added that newline.